src/_locales/ja/messages.json

```diff
@@ -0,0 +1,723 @@
+    },
+    "import_user_data": {
+        "message": "ユーザーデータをインポート",
+        "description": ""
+    },
+    "options_domain_filter_dnt": {
+        "message": "EFFのDNTポリシーに準拠",
+        "description": "Dropdown control setting on the Tracking Domains options page tab."
+    },
+    "popup_help_button": {
+        "message": "ヘルプ",
+        "description": "Tooltip that comes up when you hover over the question mark button in the upper right corner of the popup."
+    },
+    "options_domain_filter_allow": {
+        "message": "許可済み",
+        "description": "Dropdown control setting on the Tracking Domains options page tab."
+    },
+    "intro_donate_heading": {
+        "message": "プライバシーはチームスポーツです!",
+        "description": "Part of the 'donate' section on the intro page."
+    },
+    "options_domain_filter_all": {
+        "message": "すべて",
+        "description": "Dropdown control setting on the Tracking Domains options page tab."
+    },
+    "report_cancel": {
+        "message": "キャンセル",
+        "description": ""
+    },
+    "tooltip_block": {
+        "message": "スライダーを左に動かすと、ドメインがブロックされます",
+        "description": ""
+    },
+    "options_enable_dnt_checkbox": {
+        "message": "ウェブサイトに \"<a href='https://globalprivacycontrol.org/' target='_blank'>Global Privacy Control</a>\" と \"<a href='https://www.eff.org/issues/do-not-track' target='_blank'>Do Not Track</a>\" 信号を送信する",
+        "description": "Checkbox label for enabling/disabling the Sec-GPC and DNT signals, found on the general settings page"
+    },
+    "options_disable_google_nav_error_service": {
+        "message": "アクセスしたウェブアドレスをGoogleに送信しないようにします。これにより、ページが見つからないときに類似のページを提案することが無効になります。",
+        "description": "(Chrome only) Checkbox label found on the general settings page"
+    },
+    "options_disable_hyperlink_auditing": {
+        "message": "クリックしたリンクを追跡されないようにする  (\"hyperlink auditing\")",
+        "description": "Checkbox label found on the general settings page"
+    },
+    "options_disable_network_prediction": {
+        "message": "プリフェッチを無効にする",
+        "description": "Checkbox label found on the general settings page"
+    },
+    "options_disable_floc": {
+        "message": "Federated Learning of Cohorts (FLoC)を無効化する",
+        "description": "(Chrome only) Checkbox label on the general settings page."
+    },
+    "options_domain_filter_user": {
+        "message": "ユーザー定義",
+        "description": "Dropdown control setting on the Tracking Domains options page tab."
+    },
+    "popup_info_firstparty_protections": {
+        "message": "リンクトラッキングプロテクションを有効にする",
+        "description": "Clickable status header text in the popup. Shown when Privacy Badger's link tracking protection is active (for example, on facebook.com)."
+    },
+    "popup_info_firstparty_description": {
+        "message": "一部のウェブサイトでは、$LINK_START$あなたがリンクをクリックしてウェブサイトを離れるたびに、リンクトラッキングを使用してあなたを追跡しています。$LINK_END$ Privacy Badgerは、このウェブサイト上のそのようなリンクからトラッキングを削除します。",
```

Comment:
Just checking, is `$LINK_END$` in the appropriate place?

Reply:
OK 
I think there is no problem.

Reply:
The period at the end of the sentence is part of the link. Do we want that?

Reply:
Hmm...
Certainly the label on this link seems too long.
I will improve it!

Reply:
Everything between `$LINK_START$` and `$LINK_END$` will be a hyperlink. Does the translation say something like, "use link tracking to follow you" as the link text? That is what the original message says.

Reply:
OK!
I'll fix it to be that way.